model_analyzer/reports/report_manager.py

```diff
@@ -461,12 +458,11 @@ def _build_summary_table(self,
                 row = [
                     model_config.get_field('name'),
                     model_config.dynamic_batching_string(), instance_group_str,
-                    measurement.get_metric('perf_latency').value(),
-                    measurement.get_metric('perf_throughput').value(),
-                    measurement.get_metric('cpu_used_ram').value(),
-                    measurement.get_metric('gpu_used_memory').value(),
-                    round(
-                        measurement.get_metric('gpu_utilization').value(), 1)
+                    measurement.get_metric_value('perf_latency'),
+                    measurement.get_metric_value('perf_throughput'),
+                    measurement.get_metric_value('cpu_used_ram'),
```

Comment:
If there is no `cpu_used_ram` will the report still work fine?

Reply:
Yes. It will just return `0` by default. The behavior is similar to running the CPU monitor under remote mode.

Reply:
There is currently very few tests that check the reports. It would be a good idea to generate two examples and link them in this PR so that we can look at them.